packages/react-storage/src/components/StorageBrowser/components/PaginationButton.tsx

```diff
@@ -7,10 +7,12 @@ import {
   InputElementProps,
 } from '../context/elements';
 
+type ButtonTypes = 'previous' | 'next';
```

Comment:
Nit: This is probably my fault but I think this should be singular `ButtonType`

Reply:
Oh yeah, I didn't give that a second thought. Updated